### DISCUSSION TITLE
about telegraf.

### DISCUSSION THREAD

**Discussion by qigeqigeqi** [apache/skywalking#10831] [Q&A] [answered]
How to access the data reported by telegraf, but did not find the relevant reported interface information?

**Comment by wu-sheng** [comment, marked as answer]
Follow the docs, and it was added 9.3.0.

https://skywalking.apache.org/docs/main/v9.4.0/en/setup/backend/telegraf-receiver/

**Reply by qigeqigeqi** [reply]
From the information obtained on this page, I still can't know which port Skywalking uses to collect the data reported by telegraf, so I don't know which port to configure in telegraf.

**Reply by wu-sheng** [reply]
Read application.yml telegraf receiver is there.

**Reply by qigeqigeqi** [reply]
I configured hostname:12800/telegraf to report data under telegraf.conf, and I can see the virtual machine node in the basic service in the UI, but there is no relevant data indicator in the virtual machine node.
